tests/regression/issue2172.bats

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bats
+
+# bats file_tags=regression,bin:init
+
+setup() {
+  load "../common/lib"
+  load "../common/lib/env"
+  env.setup
+  env.init "dev" "baremetal"
+  common_setup
+}
+
+teardown() {
+  env.teardown
+}
+
+@test "issue 2172 has not regressed" {
+  run ck8s init both
```

Comment:
This is not needed anymore because `env.init` runs it.

Reply:
Removed, thanks!